src/main/java/net/dv8tion/jda/core/audit/AuditLogEntry.java

```diff
@@ -108,7 +122,7 @@ public Guild getGuild()
      * The {@link net.dv8tion.jda.core.entities.User User} responsible
      * for this action.
      *
-     * @return The User instance
+     * @return Possibly-null User instance
      */
     public User getUser()
```

Comment:
Maybe we should add an `@Nullable` to these

Reply:
Addition of the `@Nullable` annotation breaks pre-existing kotlin code that assumes not-null.

Reply:
Bad code to assume things it shouldn't be assuming.